templates/glauth.toml.j2

```diff
@@ -0,0 +1,28 @@
+#################
+# General configuration.
+debug = true
+
+#################
+# Server configuration.
+[ldap]
+enabled = true
+listen = "0.0.0.0:{{ ldap_port }}"
```

Comment:
Okay for now - likely want to have network bindings to ensure that this server is only listening on the networks that it should be listening to.